core/src/main/scala/ai/h2o/sparkling/backend/utils/ArgumentBuilder.scala

```diff
@@ -28,7 +28,9 @@ class ArgumentBuilder() {
   }
 
   def add(args: Seq[String]): ArgumentBuilder = {
-    arguments = arguments ++ args
+    if (args.nonEmpty) {
```

Comment:
nit: Does this condition have any effect?

Reply:
Yup -> https://github.com/h2oai/sparkling-water/blob/c5b2e075ef28f86e990fd24db0de45df81b75a8d/core/src/main/scala/ai/h2o/sparkling/backend/external/ExternalH2OBackend.scala#L110

Reply:
Ah, I see. Good catch, actually no :D

arguments ++ Seq() is basically just arguments

Reply:
will remove